frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/filter/filter_info.dart

```diff
@@ -23,9 +23,8 @@ class FilterInfo {
   }
 
   DateFilterPB? dateFilter() {
-    if (filter.fieldType != FieldType.DateTime &&
-        filter.fieldType != FieldType.UpdatedAt &&
-        filter.fieldType != FieldType.CreatedAt) {
+    if ([FieldType.DateTime, FieldType.UpdatedAt, FieldType.CreatedAt]
+        .contains(filter.fieldType)) {
```

Comment:
Should be `!` at the beginning, so it does not contains the fieldType if I'm not wrong

Reply:
You are right, Thanks.